README.md

```diff
@@ -65,6 +65,31 @@ $ ufmt diff <path> [<path> ...]
 ```
 
 
+`pre-commit` hook
+-----------------
+
+µfmt provides a [`pre-commit`](https://pre-commit.com) hook. To format your diff before 
+every commit, add the following to your `.pre-commit-config.yaml` file:
+
+```yaml
+  - repo: https://github.com/omnilib/ufmt
+    rev: 1.3.0
+    hooks:
+      - id: ufmt
+```
+
+You can change the `rev` to any version `>= 1.3.0`. To pin `black` and `usort`, use the 
```

Comment:
I just assumed this is the next release. Let me know if you rather want to release `1.2.2` or `2.0.0` instead.